packages/devtools_app/lib/src/shared/diagnostics/generic_instance_reference.dart

```diff
@@ -19,11 +19,14 @@ import 'diagnostics_node.dart';
 /// Sentinel, or primitive type.
 class GenericInstanceRef {
   GenericInstanceRef({
+    this.expandType = ExpandType.members,
```

Comment:
instead of having adding expandType to GenericInstanceRef, why not create a new class `ObjectReferences extends GenericInstanceRef`? then below, instead of checking `expandType != ExpandType.members` (which feels a bit brittle), you can check the type: `variable.ref == ObjectReferences`.